frame/aura/README.md

```diff
@@ -1,6 +1,6 @@
 # Aura Module
 
-- [`aura::Trait`](https://docs.rs/pallet-aura/latest/pallet_aura/trait.Trait.html)
+- [`aura::Config`](https://docs.rs/pallet-aura/latest/pallet_aura/trait.Config.html)
```

Comment:
these are triggering the CI error

Reply:
yes but I guess it is more correct than before

Reply:
These readme files are/should be regnerated on the next release. So it should not be required that you change this manually.